GSA/digitalgov.gov
# update hugo

## Summary

Updated hugo to 1.133.0

CircleCI now uses the .hugo-version file to determine what version of hugo to build with. 

## Preview
Click the link below to see CircleCI now using the latest version of hugo. 


[Link to Preview](https://app.circleci.com/pipelines/github/GSA/digitalgov.gov/21571/workflows/2048945b-c6c0-470c-8027-97d716e8eebd/jobs/60622/parallel-runs/0/steps/0-110)

![image](https://github.com/user-attachments/assets/4969f573-27d2-4c06-a626-98b93a9a539e)




## Solution

Provide a summary of the solution this PR offers.




## How To Test

1. Build Hugo Locally
2. [Use CircleCI CLI To Test build](https://circleci.com/docs/how-to-use-the-circleci-local-cli/#run-a-job-in-a-container-on-your-machine). (Or make a draft pr.)
3. Watch build logs to ensure version specified in .hugo-version is being used. 




## Dependency updates

| Dependency name              | Previous version | New version |
| ---------------------------- | :--------------: | :---------: |
| [Hugo](https://github.com/gohugoio/hugo/releases/tag/v0.133.1)  |     [0.123.3]      |   [0.133.1]   |
